app/src/main/res/layout/fragment_search.xml

```diff
@@ -6,13 +6,46 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <LinearLayout
+        android:id="@+id/correct_suggestion_panel"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignTop="@id/error_panel"
+        android:layout_marginTop="0dp"
+        android:orientation="horizontal">
+
+        <TextView
+            android:id="@+id/correct_suggestion_helper_text"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="10dp"
+            android:layout_marginTop="10dp"
+            android:layout_marginBottom="10dp"
+            android:textSize="@dimen/search_suggestion_helper_text_size"
+            android:textStyle="normal"
+            tools:text="@string/did_you_mean" />
+
+        <TextView
+            android:id="@+id/correct_suggestion_text"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
+            android:layout_marginStart="5dp"
+            android:layout_weight="1"
+            android:textColor="@color/background_title_color"
+            android:textSize="@dimen/search_suggestion_text_size"
+            android:textStyle="bold|italic"
+            tools:text="z101bara" />
```

Comment:
Why? We pretty much never use `tools:text`, and "Did you mean: z101bara" is of course bs, as "z101bara" is the incorrect search query, which it would obviously never suggest.

Reply:
Haha, I know it's BS, it was written for the sake of preview in the Android Studio designer. Would you like me to remove it?

Reply:
If you're going to put something there, do something that's not BS such as "newpipe"